### PR TITLE
fix(@desktop/communities): Close popup when edit is successful

### LINKED ISSUE #4126: Community description is not updated immediately
# Bug Report

## Description

When editing community description, it is not updated immediately, u need to close the modal and re-open it to see the changes.

## Steps to reproduce

1. create a community
2. open community profile
3. edit
4. edit community description and save

#### Expected behavior

About section of community is updated immediately

#### Actual behavior

New value appears only after re-opening pop up

https://user-images.githubusercontent.com/82375995/142202198-f90dc7a8-63f4-4b4c-9899-703c4f97252e.mov

### Additional Information

- Status desktop version: master, commit `663fea0272841ee56dd7ca11899a90762fb818ec`
- Operating System: macOS, BigSur

### PR DESCRIPTION
**Pull request by alaibe**
fixes #4126

### What does the PR do

When saving the communities all the popup are closed

### Affected areas

Create/Edit Community popup

### CODE REVIEW

File: ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml [hunk 1]
```diff
@@ -21,6 +21,7 @@ StatusModal {
     property var store
     property bool isEdit: false
     property QtObject community: popup.store.chatsModelInst.communities.activeCommunity
+    property var onSave: () => {}
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml:24]: You could use a `signal` instead. It's a little cleaner

**Comment by anastasiyaig** [comment]
as discussed offline, i am not clear on the reason, but community creation hangs the app in this PR forever, maybe rebase required or something, please dont merge yet

**Review by jrainville — APPROVED** [review]

File: ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml (hunk 1 above)
Comment (alaibe) [reply ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml:24]: Discussed offline, couldn't make it work as we want :(

Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml:24]: I have the same observation, could you in a brief explain please?

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**